tailwind.config.js

```diff
@@ -1,7 +1,12 @@
 module.exports = {
-  content: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
+  content: ["./index.html", "./src/**/*.{vue,js,ts,jsx,tsx}"],
```

Comment:
Why replace the single quote ?

Reply:
C'est notre config ESLint qui veut ça, rien de bien particulier ça me parait bon vu que ça permet d'avoir tout partout pareil